webextensions/api/history.json

```
@@ -112,7 +112,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/addUrl",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```

:x: https://github.com/chromium/chromium/commit/bb0a6a0af3be6d24a5c8f9ebd2e686347a5c882d in [5.0.368.0](https://github.com/chromium/chromium/releases/tag/5.0.368.0) before 5.0.375.

```suggestion
                "version_added": "5"
```